src/readme.js

```diff
@@ -0,0 +1,199 @@
+const buildTitleAndBadge = ({ title, license }) => `# ${title}
+![${license}](https://img.shields.io/static/v1?label=${license}&message=License&color=green)`;
+
+const buildTableOfContents = ({ installationIncluded, testIncluded }) => {
+  if (installationIncluded && testIncluded) {
+    return `
+
+## Table of contents
+1. [Description](#description)
+2. [User Story](#user-story)
+3. [User Guidance](#guidance)
+    1. [Usage](#usage)
+    1. [Installation Instructions](#installation)
+    2. [Test Instructions](#test)
+    3. [Contribution Guidelines](#contribution)
+4. [Questions](#questions)
+
+`;
+  }
+  if (installationIncluded && !testIncluded) {
+    return `
+
+## Table of contents
+1. [Description](#description)
+2. [User Story](#user-story)
+3. [User Guidance](#guidance)
+    1. [Usage](#usage)
+    1. [Installation Instructions](#installation)
+    2. [Contribution Guidelines](#contribution)
+4. [Questions](#questions)
+
+`;
+  }
+  if (!installationIncluded && testIncluded) {
+    return `
+
+## Table of contents
+1. [Description](#description)
+2. [User Story](#user-story)
+3. [User Guidance](#guidance)
+    1. [Usage](#usage)
+    1. [Test Instructions](#test)
+    2. [Contribution Guidelines](#contribution)
+4. [Questions](#questions)
+
+`;
+  } else {
+    return `
+
+## Table of contents
+1. [Description](#description)
+2. [User Story](#user-story)
+    1. [Usage](#usage)
+    3. [User Guidance](#guidance)
+    1. [Contribution Guidelines](#contribution)
+4. [Questions](#questions)
+
+`;
+  }
+};
+
+const buildDescription = ({ description, userStory }) => `
+<a id="description"></a>
+### Description of application 
+${description}
+
+<a id="user-story"></a>
+### User Story 
+
+${userStory}
+`;
+
+const buildInstructionsList = (instructionsArray) => {
+  const list = instructionsArray.join("\n -");
+  return ` -${list}`;
+};
+
+const buildGuidance = (
+  { installationIncluded, testIncluded, usage, contribution },
+  instructionsArray,
+  testArray
+) => {
+  const instruct = buildInstructionsList(instructionsArray);
+  const test = buildInstructionsList(testArray);
+
+  if (installationIncluded && testIncluded) {
```

Comment:
Again, this function needs a major refactor to avoid all of these repeated `if` `else` sections. Can you break these down into separate functions - one that looks at installation and returns the required html if it exists, and one to look at the tests? 

As far as I can see, the `usage` and `guidance` sections are always included - if they're not dependant on the installation/test sections, why don't we just add these separately to the generated readme?